MapboxNavigation/NavigationViewController.swift

```diff
@@ -403,6 +408,18 @@ public class NavigationViewController: UIViewController {
         let secondsRemaining = routeProgress.currentLegProgress.currentStepProgress.durationRemaining
 
         mapViewController?.notifyDidChange(routeProgress: routeProgress, location: location, secondsRemaining: secondsRemaining)
+
+        if enableNightStyleInsideTunnelFeatureEnabled,
+            let currentIntersection = routeProgress.currentLegProgress.currentStepProgress.currentIntersection,
+            let classes = currentIntersection.outletRoadClasses {
+                if classes.contains(.tunnel) {
+                    styleManager.applyStyle(type:.nightStyle)
+                    routeController.isDeadReckoningEnabled = true
+                } else {
+                    styleManager.timeOfDayChanged()
+                    routeController.isDeadReckoningEnabled = false
```

Comment:
Why is `isDeadReckoningEnabled` being mutated?

Reply:
There is no need to continue mark the intersection as a candidate for dead reckoning.